### PR TITLE
Improve signature popup window

### PR DESCRIPTION
**Pull request by girishji**
Add 1 character padding to the sides of signature popup window. Otherise, the leftmost character of popup looks like continuation of line from underneath text in some colorschemes. A subtle improvemnt.

M  autoload/lsp/signature.vim

### CODE REVIEW

**Comment by girishji** [comment]
[Correctly handle keys inside symbol popup window](https://github.com/yegappan/lsp/pull/448/commits/aa3777fa2317eedbad00d2b9df73864be7099d87) 

Should have been a separate PR. Didn't know it will just merge 2 commits.

File: autoload/lsp/symbol.vim [hunk 1]
```diff
@@ -811,33 +811,48 @@ def SymbolMenuFilterKey(symPopupMenu: number,
...
+      || key == "\<ScrollWheelUp>"
+    var ln = getcurpos(symPopupMenu)[1]
+    win_execute(symPopupMenu, "normal! k")
+    if ln == getcurpos(symPopupMenu)[1]
+      win_execute(symPopupMenu, "normal! G")
+    endif
+  elseif key == "\<PageDown>"
+    win_execute(symPopupMenu, 'normal! ' .. "\<C-d>")
+  elseif key == "\<PageUp>"
+    win_execute(symPopupMenu, 'normal! ' .. "\<C-u>")
```
Comment (yegappan) [inline autoload/lsp/symbol.vim:843]: Can you use double quotes for the entire string and avoid the string concatenation ".."?

File: autoload/lsp/symbol.vim [hunk 2]
```diff
@@ -811,33 +811,48 @@ def SymbolMenuFilterKey(symPopupMenu: number,
...
+      || key == "\<C-p>"
+      || key == "\<Up>"
+      || key == "\<ScrollWheelUp>"
+    var ln = getcurpos(symPopupMenu)[1]
+    win_execute(symPopupMenu, "normal! k")
+    if ln == getcurpos(symPopupMenu)[1]
+      win_execute(symPopupMenu, "normal! G")
+    endif
+  elseif key == "\<PageDown>"
+    win_execute(symPopupMenu, 'normal! ' .. "\<C-d>")
```
Comment (yegappan) [inline autoload/lsp/symbol.vim:841]: Can you use double quotes for the entire string and avoid the string concatenation ".."?

**Review by yegappan — CHANGES_REQUESTED** [review]

File: autoload/lsp/symbol.vim [hunk 3]
```diff
@@ -811,33 +811,48 @@ def SymbolMenuFilterKey(symPopupMenu: number,
...
+      || key == "\<ScrollWheelDown>"
+    var ln = getcurpos(symPopupMenu)[1]
+    win_execute(symPopupMenu, "normal! j")
+    if ln == getcurpos(symPopupMenu)[1]
+      win_execute(symPopupMenu, "normal! gg")
+    endif
+  elseif key == "\<S-tab>"
+      || key == "\<C-p>"
+      || key == "\<Up>"
+      || key == "\<ScrollWheelUp>"
```
Comment (yegappan) [inline autoload/lsp/symbol.vim:834]: Can you also update the help text to describe the new keys that can be used in the symbol popup window?

**Pull request merged**